settings.py

```diff
@@ -2,8 +2,8 @@
 
 MODEL_PATH = os.environ.get("MODEL_PATH", "model/detect_mentions/")
 BACKUP_PATH = os.environ.get("BACKUP_PATH", "model/backup/")
-MDR_COMMENT_ENDPOINT_TOKEN = os.environ["MDR_COMMENT_ENDPOINT_TOKEN"]
-MDR_COMMENT_ENDPOINT = os.environ["MDR_COMMENT_ENDPOINT"]
+MDR_COMMENT_ENDPOINT_TOKEN = os.environ.get("MDR_COMMENT_ENDPOINT_TOKEN", ".credentials/coral-talk-token.json")
```

Comment:
Now the default argument of the variable is a path and the environment variable holds the token itself. This will lead to a conflict.